locales/en-US/notes.properties

```diff
@@ -80,6 +80,8 @@ syncComplete2=Synced at {time}
 # LOCALIZATION NOTE (mergeWarning): This string is displayed in case of a merge conflict during a note sync.
 # It is displayed below the 'synced' notes and above the 'local' notes. It is a visual separator.
 mergeWarning=**** On this computer: ****
+syncToMail=Sync to {mail}
```

Comment:
Please add a LOCALIZATION NOTE explaining what {mail} is about. (see the mergeWarning LOCALIZATION NOTE above.